data/tooltips.txt

```diff
@@ -938,12 +938,24 @@ tip "no thruster!"
 tip "no steering!"
 	`This ship does not have any steering systems installed. If it took off, it would be unable to turn.`
 
+tip "no energy?"
+	`This ship does not have any source of energy to power its systems. Although it can take off while docked on an other ship, it will not function once deployed.`
```

Comment:
```suggestion
	`This ship does not have any source of energy to power its systems. Although it can take off while docked on another ship, it will not function once deployed.`
```